test/parallel/test-benchmark-dgram.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+require('../common');
+
+const runBenchmark = require('../common/benchmark');
+
+runBenchmark('dgram',[ ]);
+
```

Comment:
The linter should probably complain about the extra empty line, can you remove this one?